qos/cometbft/response_status.go

```diff
@@ -15,11 +15,16 @@ type (
 	// Result struct is the expected response from the `/status` endpoint.
 	// Reference: https://docs.cometbft.com/v1.0/spec/rpc/#response-1
 	Result struct {
+		NodeInfo NodeInfo `json:"node_info"`
```

Comment:
TODO_IMPROVE to reference external proto: https://github.com/cometbft/cometbft/blob/4226b0ea6ab4725ef807a16b86d6d24835bb45d4/rpc/core/types/responses.go#L103

Reply:
I just imported the struct and deleted the one defined here.